components/table/table-wrapper.js

```diff
@@ -325,17 +329,50 @@ export class TableWrapper extends RtlMixin(LitElement) {
 
 	}
 
-	_handleSlotChange(e) {
+	/* required by PageableMixin */
```

Comment:
The implementation seems generic here, but the implementation required for lists is different.